inference_perf/config.py

```diff
@@ -37,12 +37,17 @@ class LoadType(Enum):
     POISSON = "poisson"
 
 
-class LoadConfig(BaseModel):
-    type: LoadType = LoadType.CONSTANT
+class LoadStage(BaseModel):
     rate: int = 1
     duration: int = 1
 
 
+class LoadConfig(BaseModel):
```

Comment:
Its not immediately apparent if the tool waits for all requests to be resolved before it starts sending requests in the next stage, could we either call out the default behavior and/or make whether or not the tool does this configurable?

Reply:
Looks like the time to wait between runs is configurable which should suffice for now.